cmd/mesh-bootstrap/main.go

```diff
@@ -172,7 +172,12 @@ func main() {
 		newDHT = func(h host.Host) (routing.PeerRouting, error) {
 			var err error
 			dhtDir := getDHTDir(config)
-			kadDHT, err = p2p.NewDHT(ctx, dhtDir, h)
+			// Set up the DHT to use LevelDB.
+			store, err := leveldbStore.NewDatastore(dhtDir, nil)
+			if err != nil {
+				return nil, err
+			}
+			kadDHT, err = dht.New(ctx, h, dhtopts.Datastore(store), dhtopts.Protocols(p2p.DHTProtocolID))
```

Comment:
We needed to do this in order to continue using leveldb in the Mesh bootstrap nodes. It would be better to use our custom database solution eventually, but this would require large refactors in the `db` package. Since production bootstrap nodes use the PostgreSQL storage solution primarily and the bootstrap node is a stable implementation, I'd prefer to do this in a separate PR.

Reply:
Oskar: The bootstrap nodes use LevelDB and it's not shared.

Reply:
Thanks for following up on this. I'll remove PostgreSQL as a storage option since no one uses it.